lib/graphql/directive.rb

```diff
@@ -16,16 +16,19 @@ class Directive
       MUTATION =            :MUTATION,
       SUBSCRIPTION =        :SUBSCRIPTION,
       FIELD =               :FIELD,
+      FIELD_DEFINITION =    :FIELD_DEFINITION,
       FRAGMENT_DEFINITION = :FRAGMENT_DEFINITION,
       FRAGMENT_SPREAD =     :FRAGMENT_SPREAD,
       INLINE_FRAGMENT =     :INLINE_FRAGMENT,
+      ENUM_VALUE =          :ENUM_VALUE,
     ]
 
     def initialize
       @arguments = {}
     end
 
     def include?(arguments)
+      return true if include_proc.nil?
```

Comment:
The bad news is, I caused a conflict. 

The good news is, this is implemented directly in `DirectiveChecks` and this method is gone :tada: